etc/profile-m-z/totem.profile

```diff
@@ -8,6 +8,7 @@ include globals.local
 
 # Allow lua (required for youtube video)
 include allow-lua.inc
+include allow-python3.inc
```

Comment:
Cosmetic remark: please add a newline above and our usual comment for Python support to keep syntax in sync with our other profiles.

# Allow python (blacklisted by disable-interpreters.inc)

Reply:
> I think it works becuase allow-python3.inc "noblacklists" the python library paths, which are blacklisted by disable-interpreters.inc.

Sure, that's what including allow-python3.inc achieves. But I'm wondering about the python console plugin. IMO that needs private-bin `python*,totem`. Do you have that plugin enabled? While we're on the subject, which totem plugins are enabled on your side, if any?